### DISCUSSION TITLE
How do I edit the icons at the bottom of about page?

### DISCUSSION THREAD

**Discussion by wcmohr** [alshedivat/al-folio#1697] [Q&A] [answered]
I removed the blog page and now I'm not sure how to remove the icon at the bottom of the about page that links to the blog.  

The link with the dot and quarter circles gives me this html page:

This XML file does not appear to have any style information associated with it. The document tree is shown below.
<feed xmlns="http://www.w3.org/2005/Atom" xml:lang="en">
<generator uri="https://jekyllrb.com/" version="4.3.2">Jekyll</generator>
<link href="https://wcmohr.github.io/feed.xml" rel="self" type="application/atom+xml"/>
<link href="https://wcmohr.github.io/" rel="alternate" type="text/html" hreflang="en"/>
<updated>2023-09-02T15:05:58+00:00</updated>
<id>https://wcmohr.github.io/feed.xml</id>
<title type="html">blank</title>
<subtitle>A jupyter notebook a day keeps the cobwebs at bay. </subtitle>
<entry>

Any advice would be great!

**Comment by george-gca** [comment]
In [_pages/about.md](https://raw.githubusercontent.com/alshedivat/al-folio/master/_pages/about.md), simply set in the front matter `latest_posts: false`.

**Reply by wcmohr** [reply]
This is already set as false.  I do have this specified:
social: true  # includes social icons at the bottom of the page.    I'm wondering how do I remove the RSS Feed icon but keep the linkedin and email and github icons.

**Reply by george-gca** [reply, marked as answer]
Ahhhh now I get it.

Set `rss_icon: false` in [_config.yml](https://github.com/alshedivat/al-folio/blob/81b89c8d454c846eb5b88ccfd2bbf2a556d7ac0b/_config.yml#L48)
